package.json

```diff
@@ -53,6 +53,7 @@
     "glsl-noise": "^0.0.0",
     "lodash.omit": "^4.5.0",
     "lodash.pick": "^4.4.0",
+    "open-simplex-noise": "^2.4.0",
```

Comment:
`three` has a a simplex noise function - `three/examples/jsm/math/SimplexNoise` can we not use that?

Reply:
Ah cool I wasn't aware of that, should be able to swap to that.

Reply:
only if you think it'll work? I haven't looked at it in depth and tbf it isn't documented in THREE at all (surprised? no).

Reply:
Assuming it's some type of gradient noise function, should be easy enough to swap in. I can look into it a bit later today.